bin/setup

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+yarn install
+
+if [[ ! -r ./.env ]]; then
+  echo "Creating config file..."
+  echo "API_KEYS=foo,bar"
+  echo "PORT=3333" >> .env
+  if [[ -x $(which docker) ]]; then
+    read -n 1 -r -p "Do you want to use docker to run the DB? [y/n] "
```

Comment:
Are we happy to assume there's a postgres DB being used?

Reply:
If so, shall we include a basic docker-compose file to get it up and running and a .env.example file?